egs2/TEMPLATE/asr1/db.sh

```diff
@@ -190,6 +190,7 @@ KIRITAN=
 NAMINE=
 KATHBATH=downloads
 GRAMVAANI=downloads
+WAV2GLOSS=downloads
```

Comment:
Please add in the egs2/README.md

Reply:
Sorry, missed this, will fill up now!

Reply:
Fixed in 16d7f201772d4c50ab64311353f7f4bf7bff6ed1